test/src/639-specialfiles/main

```diff
@@ -1,6 +1,7 @@
 cvmfs_test_name="Handling special files"
 
 export CVMFS_TEST_UNIONFS="overlayfs"
+export CVMFS_IGNORE_SPECIAL_FILES="true"
```

Comment:
I will try to refactor this test case a little bit and add more tests into it. I will do it in separate commits which can be added to this PR before it's merged.